docs/user/rstudio/readme.md

```diff
@@ -0,0 +1,46 @@
+# RStudio User Guide
+
+-   Source code: <https://github.com/rstudio/rstudio/tree/main/docs/user/rstudio>
+-   Public URL: <https://docs.rstudio.com/ide/user/>
+
+> If you’re interested in getting involved with open source, contributing to documentation is a good—and helpful—place to start. - [James Turnbill](https://increment.com/documentation/documentation-as-a-gateway-to-open-source/)
+
+If you have found issues with the RStudio User Guide, please check for existing duplicate issues/pull requests. 
+If a matching Issue/Pull Request doesn't exist, please then open an [Issue](https://github.com/rstudio/rstudio/issues/new/choose) or Pull Request. 
```

Comment:
Is "Pull Request" intended to be a link, such as `https://github.com/rstudio/rstudio/pulls` ?

Reply:
The PR workflow would be via "Edit this page" or a more manual process as opposed to the direct link to an issue (I wanted to very very specific link to the "choose" an issue type).